docs/sources/setup-grafana/configure-docker.md

```diff
@@ -60,48 +93,281 @@ The following settings are hard-coded when launching the Grafana Docker containe
 
 - Admin password secret: `/run/secrets/admin_password`
 - Environment variable: `GF_SECURITY_ADMIN_PASSWORD__FILE=/run/secrets/admin_password`
 
-## Configure AWS credentials for CloudWatch Support
+### Configure Docker secrets credentials for AWS CloudWatch
+
+Grafana ships with built-in support for the [Amazon CloudWatch datasource]({{< relref "../datasources/aws-cloudwatch" >}}). To configure the data source, you must provide information such as the AWS ID-Key, secret access key, region, and so on. You can use Docker secrets as a way to provide this information.
+
+Example:
+
+The example below shows how to use Grafana environment variables via Docker Secrets for the AWS ID-Key, secret access key, region, and profile.
+
+The example uses the following values for the AWS Cloudwatch data source:
 
 ```bash
-docker run -d \
--p 3000:3000 \
---name=grafana \
--e "GF_AWS_PROFILES=default" \
--e "GF_AWS_default_ACCESS_KEY_ID=YOUR_ACCESS_KEY" \
--e "GF_AWS_default_SECRET_ACCESS_KEY=YOUR_SECRET_KEY" \
--e "GF_AWS_default_REGION=us-east-1" \
-grafana/grafana-enterprise
+AWS_default_ACCESS_KEY_ID=aws01us02
+AWS_default_SECRET_ACCESS_KEY=topsecret9b78c6
+AWS_default_REGION=us-east-1
 ```
 
-You may also specify multiple profiles to `GF_AWS_PROFILES` (e.g.
-`GF_AWS_PROFILES=default another`).
+1. Create a Docker secret for each of the values noted above.
+
+   ```bash
+   echo "aws01us02" | docker secret create aws_access_key_id -
+   i4g62kyuy80lnti5d05oqzgwh
+   ```
+
+   ```bash
+   echo "topsecret9b78c6" | docker secret create aws_secret_access_key -
+   uegit5plcwodp57fxbqbnke7h
+   ```
+
+   ```bash
+   echo "us-east-1" | docker secret create aws_region -
+   fxbqbnke7hplcwodp57fuegit
+   ```
+
+1. Run the following command to determine that the secrets were created.
 
-Supported variables:
+   ```bash
+   $ docker secret ls
+   ```
+
+   The output from the command should look similar to the following:
+
+   ```
+   ID                          NAME           DRIVER    CREATED              UPDATED
+   i4g62kyuy80lnti5d05oqzgwh   aws_access_key_id             5 minutes ago        5 minutes ago
+   uegit5plcwodp57fxbqbnke7h   aws_secret_access_key         3 minutes ago        3 minutes ago
+   fxbqbnke7hplcwodp57fuegit   aws_region                    About a minute ago   About a minute ago
+   ```
+
+   Similarly, you can also find more information about the above created secrets using the `docker inspect` command.
+
+   For example, to inspect the `aws_access_key_id` secret, run the following command:
+
+   ```bash
+   $ docker inspect aws_access_key_id
+   [
+       {
+           "ID": "i4g62kyuy80lnti5d05oqzgwh",
+           "Version": {
+               "Index": 16
+           },
+           "CreatedAt": "2023-05-29T21:43:37.247825144Z",
+           "UpdatedAt": "2023-05-29T21:43:37.247825144Z",
+           "Spec": {
+               "Name": "aws_access_key_id",
+               "Labels": {}
+           }
+       }
+   ]
+   ```
```

Comment:
I don't understand the purpose of this section here. What do we mean by 'more information'? Is it important in the overall completion of this task? If not, I recommend removing this content.

Reply:
I have now removed this section